pkg/core/annotations.go

```diff
@@ -20,9 +20,11 @@ import (
 	"strconv"
 
 	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/knative/serving/pkg/reconciler/v1alpha1/configuration/resources"
```

Comment:
We shouldn't use this package, it's an implementation detail for serving

Reply:
hmm, I agree that it is not ideal, but do you have a better idea how to access the annotations of the build definition embedded in the kservice?

Reply:
Do the equivalent of https://github.com/knative/serving/blob/99ceee84e0c158ea1388af3420feff63b37f60ef/pkg/reconciler/v1alpha1/configuration/resources/build.go#L62-L96 yourself.

If this is something we want Knative to maintain, we should add it as a method in https://github.com/knative/serving/tree/master/pkg/apis/serving/v1alpha1

Reply:
That seems worse, at least the implementation in knative is more likely to be maintained even if it is not exposed